qiskit/quantum_info/operators/tolerances.py

```diff
@@ -0,0 +1,97 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Tolerances mixin class.
+"""
+
+import warnings
+from abc import ABCMeta
+
+from qiskit.exceptions import QiskitError
+from qiskit.quantum_info.operators.predicates import ATOL_DEFAULT, RTOL_DEFAULT
+
+
+class TolerancesMeta(ABCMeta):
```

Comment:
Is there a reason to use `ABCMeta` here? nothing about this looks abstract, so just wonder whether this needs to inheret from that or if we can just drop this and just use `class ToleranceMeta:`

Reply:
I think it's unnecessary now. It was probably there before because the linter complained when the old class was in the abstract BaseOperator class